RecoTracker/TransientTrackingRecHit/python/TTRHBuilders_cff.py

```diff
@@ -7,6 +7,7 @@
 from RecoTracker.TkSeedingLayers.TTRHBuilderWithoutAngle4PixelPairs_cfi import *
 from RecoTracker.TkSeedingLayers.TTRHBuilderWithoutAngle4PixelTriplets_cfi import *
 #TransientTRH builder with template
+from RecoLocalTracker.Phase2TrackerRecHits.Phase2StripCPEESProducer_cfi import *
```

Comment:
while cross-checking where exactly the SiStripCPE is included,
I would suggest to have the following include as well
`from RecoLocalTracker.SiStripRecHitConverter.StripCPEESProducer_cfi import *`
(for completeness / consistency, let say)

Reply:
@mtosi, I'd really rather not clutter this file with other useless inclusions... on the other hand I am happy to put this `import` elsewhere if you find out where the rest of the CPE ESproducer get included in the tracking chain via the existing `import`s in this file

Reply:
@mtosi 
please clarify/confirm that it's OK to go as is in this PR

Reply:
> @mtosi
> please clarify/confirm that it's OK to go as is in this PR

Reply:
+1, but I think it would be worth to have it as well (in this way it is standalone)

Reply:
uhm, after staring at this and the  testTrackRefit_phase2.py  in https://gist.github.com/mmusich/1aa32f5f0878ee514bb135ccc02407fb for a bit, I think that it's more practical to 
add to RecoTracker/TrackProducer/python/TrackRefitters_cff.py
```python
from RecoLocalTracker.Configuration.RecoLocalTracker_cff import *
``` 
the local reco definitions seem to be quite relevant there in general.
This should cover the point from @mtosi

Reply:
Thanks for the suggestion. This is implemented now in https://github.com/cms-sw/cmssw/pull/32178/commits/f17a5e50ac3bbaa994d8e22d52791e6a761b4e9e, and tested to work.